### PR TITLE
Finsh User page

### PR DESCRIPTION
**Pull request by wangwenyue**
User page info panel done.

User joined club list done.

NavBar for homepage done.

Add all function needed in user page.

### CODE REVIEW

File: src/components/BottomNav/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+const NavBarItem = ({className = '', svg, itemName = '', callback}) => (
+    <div className={`${className} navbar-item`} onClick={callback}>
+        {svg}
+        <div>{itemName}</div>
+    </div>
+);
+
+@inject('navStore') @observer
+class BottomNav extends React.Component {
```
Comment (Keith-CY) [inline src/components/BottomNav/index.js:16]: use pure function component

File: src/containers/Activity/index.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 import {observer, inject} from 'mobx-react';
 
-@inject('eventStore') @observer
-class Event extends React.Component {
+@inject('activityStore') @observer
+class Activity extends React.Component {
```
Comment (Keith-CY) [inline src/containers/Activity/index.js:5]: use pure function component

File: src/containers/Home/index.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import {observer, inject} from 'mobx-react'
+import {
+    Flex,
+} from 'antd-mobile'
+import {Header, BottomNav} from '../../components'
+import './home.css'
+
+@inject('personalStore') @observer
+class Home extends React.Component {
```
Comment (Keith-CY) [inline src/containers/Home/index.js:10]: use pure function component

File: src/containers/Personal/ClubListItem/clublistitem.css [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+.club-list-name {
+    padding-top: 15px;
+    padding-left: 15px;
+}
+
+.club-list-name span {
+    width: 105px;
+    height: 15px;
+    font-family: PingFangSC;
```
Comment (Keith-CY) [inline src/containers/Personal/ClubListItem/clublistitem.css]: PingFang only works on MacOS, add fallback fonts for Windows and mobile.

File: src/containers/Personal/ClubListItem/clublistitem.css [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+.club-list-item-right {
+    width: 20.8%;
+    height: 82px;
+    padding-top: 16.5px;
+}
+
+.club-list-item-right span {
+    width: 39px;
+    height: 12px;
+    font-family: PingFangSC;
```
Comment (Keith-CY) [inline src/containers/Personal/ClubListItem/clublistitem.css]: add fallback fonts

File: src/containers/Personal/ClubListItem/index.js [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+    avatar: 'avatar.png',
+    clubID: 1006,
+},{
+    clubName: '加班俱乐部',
+    avatar: 'avatar.png',
+    clubID: 1006,
+},];
+
+@inject('clublistStore') @observer
+class ClubListItem extends React.Component {
```
Comment (Keith-CY) [inline src/containers/Personal/ClubListItem/index.js:9]: use pure function component

File: src/containers/Personal/PersonalPanel/index.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    Flex,
+    List,
+} from 'antd-mobile'
+import './personalPanel.css'
+
+const Item = List.Item
+const Brief = Item.Brief
+
+@inject('personalStore') @observer
+class PersonalPanel extends React.Component {
```
Comment (Keith-CY) [inline src/containers/Personal/PersonalPanel/index.js:14]: use pure function component

File: src/containers/Personal/index.js [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+import {
+    Flex,
+} from 'antd-mobile'
+import PersonalPanel from './PersonalPanel'
+import ClubListItem from './ClubListItem'
+import BottomNav from '../../components/BottomNav'
+import './personal.css'
+
+@inject('personalStore') @observer
+class Personal extends React.Component {
```
Comment (Keith-CY) [inline src/containers/Personal/index.js:13]: use pure function component

File: src/containers/Personal/personal.css [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+    background-color: #f8fafe;
+    /*border: 1px solid red;*/
+}
+
+.personal-panel-club-banner {
+    padding-left: 15px;
+    padding-top: 20px;
+    /*height: 15px;*/
+    /*width: 75px;*/
+    font-family: PingFangSC;
```
Comment (Keith-CY) [inline src/containers/Personal/personal.css]: add fallback fonts

File: src/stores/navStore/index.js [hunk 10]
```diff
@@ -0,0 +1,67 @@
+import {observable, action} from 'mobx'
+import React from "react";
+
+const log = console.log.bind(console, '### navStore ')
+
+const activityIcon = <svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 40 40">
```
Comment (Keith-CY) [inline src/stores/navStore/index.js]: set svg image to single file

File: src/stores/navStore/index.js [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+        this.toggleActive(event)
+    }
+
+    @action handlePersonal = (event) => {
+        log('personal button clicked')
+        this.toggleActive(event)
+    }
+
+    toggleActive = (event) => {
+        let old = document.querySelector('.active-nav-item')
```
Comment (Keith-CY) [inline src/stores/navStore/index.js]: use `const` instead of `let`

**Review by Keith-CY — CHANGES_REQUESTED** [review]

File: src/stores/activityStore/index.js [hunk 12]
```diff
@@ -1,6 +1,6 @@
 import { observable, action } from 'mobx'
 
-class EventStore {
+class ActivityStore {
```
Comment (ashchan) [inline src/stores/activityStore/index.js:3]: Is it React practice to use `camelCase` for file/folder names (`activityStore` in this case)?

Note this is only a question.

Comment (wangwenyue) [reply src/stores/activityStore/index.js:3]: The answer is YES.

Comment (ashchan) [reply src/stores/activityStore/index.js:3]: Thanks. Then another question is, why subfolders under `components` and `containers` are capitalized `CamelCase`?

Comment (Keith-CY) [reply src/stores/activityStore/index.js:3]: In react project, file/dir name should be named in `CamelCase`, except the `Component` should be capitalized.

https://reactjs.org/docs/jsx-in-depth.html

https://stackoverflow.com/questions/30373343/reactjs-component-names-must-begin-with-capital-letters

**Comment by wangwenyue** [comment]
After discussion, no need to modify pure component for now.

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**